### ISSUE TITLE
Remove <meta name="fragment" content="!"> tags

### ISSUE THREAD

**Issue by joeyAghion** [artsy/force#104]
Pages like https://www.artsy.net/art-basel/browse/artist/mary-kelly include this in their markup to indicate that the page is significantly rendered on the client-side:

```
<meta name="fragment" content="!">
```

As a result, search engine crawlers make an additional request to https://www.artsy.net/art-basel/browse/artist/mary-kelly?_escaped_fragment_= with the goal of retrieving pre-rendered HTML. However, the page is not sitemapped, so won't be pre-rendered.

Instead, the page should either exclude the `meta` tag or be sitemapped [and pre-rendered]. I'm not sure if this affects other similar pages.

----

Title edited from: "Pages such as /art-basel/browse/artist/mary-kelly are unsitemapped but include the fragment meta tag"

**Comment by alloy** [comment]
@craigspaeth Can you have a think about this and what we should be doing. @broskoski has some thoughts on this, but would love to hear from you first.

**Comment by craigspaeth** [comment]
Going to close this issue as it's quite old now, but I'll leave my thoughts on it.

Generally speaking, I'm a proponent of trying to serve content to Google without Reflection as more and more of the web does client-side rendering and Google announced [deprecating escaped fragment](https://webmasters.googleblog.com/2015/10/deprecating-our-ajax-crawling-scheme.html) a long time ago. I believe Christine has also mentioned before that it's not productive to sitemap everything under the sun—and I'd imagine sitemapping a deep link like /art-basel/browse/artist/mary-kelly is probably a waste.

So with that said, we should just remove the use of this meta tag here (and probably across the site) and call it a day. However, that's just my opinion. #seo in Slack will have the best answers to these questions.

**Comment by joeyAghion** [comment]
I think that's reasonable, but shouldn't we remove the meta tag before closing this?

I agree that sitemaps are worth little on their own and only mentioned them here because that's how Reflection knows to crawl and pre-render the page. The current mark-up (with the meta tag but no corresponding sitemap entry) is basically preventing indexing by directing Googlebot to request [a URL that fails with `403`](https://www.artsy.net/art-basel/browse/artist/mary-kelly?_escaped_fragment_=).

**Comment by craigspaeth** [comment]
Good point Joey. I reopened the issue and changed the title to reflect next steps.
